src/domain/common/preference/preference.definition.entity.ts

```diff
@@ -36,7 +40,7 @@ export class PreferenceDefinition
   valueType!: PreferenceValueType;
 
   @Column({
-    length: SMALL_TEXT_LENGTH,
+    length: ENUM_LENGTH,
```

Comment:
nullable false?

Reply:
`false` is by default

Reply:
we have it explicit in may places, maybe also we just state it always?